From dd614d496dddf5c443a00fc1545146b40db818d9 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 25 Jan 2024 07:57:50 +0100
Subject: [PATCH] Update meson.build

---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 97371eef..e675beaf 100644
--- a/meson.build
+++ b/meson.build
@@ -2,7 +2,7 @@
 project(
     'matscipy', # Project name
     'c', 'cpp', # Project type. We need a C and C++ compiler.
-    version: run_command('python3', 'discover_version.py', check: true).stdout().strip(), # Project version
+    version: run_command('sh', '-c', 'echo ${PKG_VERSION}', check: true).stdout().strip(), # Project version
 )
 
 # https://mesonbuild.com/Python-module.html
@@ -13,4 +13,4 @@ python = pymod.find_installation('python3',
 )
 
 subdir('c')
-subdir('matscipy')
\ No newline at end of file
+subdir('matscipy')
