From 327d3316593d3ed10d521893bfdad36e7ee777cf Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 25 Jan 2024 07:41:43 +0100
Subject: [PATCH] Update meson.build

---
 matscipy/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/matscipy/meson.build b/matscipy/meson.build
index db67658a..c619783a 100644
--- a/matscipy/meson.build
+++ b/matscipy/meson.build
@@ -1,7 +1,8 @@
 # https://mesonbuild.com/Python-module.html
 
 conf_data = configuration_data()
-conf_data.set('VERSION', meson.project_version())
+cmd = run_command('sh', '-c', 'echo $PKG_VERSION')
+conf_data.set('VERSION', cmd.stdout().strip())
 version_file = configure_file(
     input: '_version.py.in',
     output: '_version.py',
